latex/ledger-spec.tex

```diff
@@ -410,21 +560,128 @@ \subsection{Witnesses}
   \begin{equation}
     \label{eq:utxo-witness-inductive}
     \inference[UTxO-wit]
-    { \var{pc} \vdash \var{utxo} \trans{utxo}{tx} \var{utxo'}\\ ~ \\
+    {
+      {
+        \begin{array}{l}
+        \var{slot}\\
+        \var{pc}
+        \end{array}
+      }
+      \vdash \var{utxo} \trans{utxo}{tx} \var{utxo'}\\ ~ \\
       & \forall i \in \txins{tx} \cdot \exists (\var{vk}, \sigma) \in \wits{\var{tx}}
       \cdot
       \mathcal{V}_{\var{vk}}{\serialised{\txbody{tx}}}_{\sigma}
       \wedge  \fun{addr_h}~{utxo}~i = \hash{vk}\\
     }
-    {\var{pc} \vdash \var{utxo} \trans{utxow}{tx} \var{utxo'}}
+    {
+      \begin{array}{l}
+        \var{slot}\\
+        \var{pc}\\
+      \end{array}
+      \vdash \var{utxo} \trans{utxow}{tx} \var{utxo'}
+    }
   \end{equation}
   \caption{UTxO with witnesses inference rules}
   \label{fig:rules:utxow}
 \end{figure}
 
-\section{Delegation}
-\label{sec:delegation}
-\input{delegation.tex}
+\begin{figure}
+  \emph{Ledger environment}
+  \begin{equation*}
+    \LEnv =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{slot} & \Slot & \text{current slot}\\
+        \var{pc} & \PrtclConsts & \text{protocol constants}\\
+      \end{array}
+    \right)
+  \end{equation*}
+  %
+  \emph{Ledger state}
+  \begin{equation*}
+    \LState =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{utxo} & \UTxO & \text{UTxO}\\
+        \var{dstate} & \DState & \text{delegation state}\\
+        \var{pstate} & \PState & \text{pool state}\\
+      \end{array}
```

Comment:
why not combine to `DWState` ?

Reply:
you are correct, it should be `DWState`, thanks!

Reply:
done